specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Computer Vision API",
+        "description": "The Computer Vision API provides state-of-the-art algorithms to process images and return information. For example, it can be used to determine if an image contains mature content, or it can be used to find all the faces in an image.  It also has other features like estimating dominant and accent colors, categorizing the content of images, and describing an image with complete English sentences.  Additionally, it can also intelligently generate images thumbnails for displaying large images effectively."
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
+        }
+    },
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+        "parameters": [
+            {
+                "$ref": "../../Common/ExtendedRegions.json#/parameters/AzureRegion"
+            }
+        ]
+    },
+    "basePath": "/vision/v1.0",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/models": {
+            "get": {
+                "description": "This operation returns the list of domain-specific models that are supported by the Computer Vision API.  Currently, the API only supports one domain-specific model: a celebrity recognizer. A successful response will be returned in JSON.  If the request failed, the response will contain an error code and a message to help understand what went wrong.",
+                "operationId": "List Domain Specific Models",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages

Reply:
Fixed